### PR TITLE
[SVCS-426] Update Style and Trivial Code Change for Tests

### PR DESCRIPTION
**Pull request by cslzchen**
I can't resist the temptation to scan through the tests and ended up refactoring style.

### CODE REVIEW

File: tests/providers/googledrive/test_provider.py [hunk 1]
```diff
@@ -2,30 +2,27 @@
 import os
 import copy
 import json
-from http import client
 from urllib import parse
+from http import HTTPStatus
```
Comment (cslzchen) [inline tests/providers/googledrive/test_provider.py]: Using `HTTPStatus` instead of `client`.

File: tests/providers/googledrive/test_provider.py [hunk 2]
```diff
@@ -231,21 +231,27 @@ class TestValidatePath:
         try:
             wb_path_v1 = await provider.validate_v1_path('/' + file_name)
         except Exception as exc:
+            wb_path_v1 = None
```
Comment (cslzchen) [inline tests/providers/googledrive/test_provider.py]: Declare and set `wb_path_v1` for integrity. (several occurrences)

File: tests/providers/googledrive/test_provider.py [hunk 3]
```diff
@@ -302,15 +309,18 @@ class TestValidatePath:
 
         current_part = parts.pop(0)
         part_name, part_is_folder = current_part[0], current_part[1]
-        name, ext = os.path.splitext(part_name)
+        _, _ = os.path.splitext(part_name)
```
Comment (cslzchen) [inline tests/providers/googledrive/test_provider.py]: For return values we don't use, use `_`. (several occurrences)

File: tests/providers/googledrive/test_provider.py [hunk 4]
```diff
@@ -550,26 +629,32 @@ class TestDelete:
     async def test_delete_root_no_confirm(self, provider):
         path = WaterButlerPath('/', _ids=('0'))
 
-        with pytest.raises(exceptions.DeleteError) as e:
+        with pytest.raises(exceptions.DeleteError) as exc:
```
Comment (cslzchen) [inline tests/providers/googledrive/test_provider.py]: Both `e` and `exc` were used in this file. I refactored this to use one consistently and I randomly choose `exc`.

File: tests/providers/googledrive/test_provider.py [hunk 5]
```diff
@@ -1555,22 +1771,24 @@ class TestCreateFolder:
...
         metadata_furl = furl.furl(provider.build_url('files', path.identifier or ''))
         metadata_url = metadata_furl.add(provider.FILE_FIELDS).url
         aiohttpretty.register_json_uri('POST', metadata_url, status=418)
 
-        with pytest.raises(exceptions.CreateFolderError) as e:
+        with pytest.raises(exceptions.CreateFolderError) as exc:
             await provider.create_folder(path)
 
-        assert e.value.code == 418
+        assert exc.value.code == 418
```
Comment (cslzchen) [inline tests/providers/googledrive/test_provider.py]: Note: neither python 3.5 or 3.6 has `418` defined. `HTTPStatus` can't be used here.

**Comment by cslzchen** [comment]
Amending commit did not trigger travis rebuild as I had expected. I will close and reopen the PR.

File: tests/providers/googledrive/test_provider.py (hunk 3 above)
Comment (TomBaxter) [inline tests/providers/googledrive/test_provider.py]: If we aren't using the output, why are we doing this at all? (two occurrences)

File: tests/providers/googledrive/test_provider.py [hunk 6]
```diff
@@ -359,18 +376,25 @@ class TestValidatePath:
 
         current_part = parts.pop(0)
         part_name, part_is_folder = current_part[0], current_part[1]
-        name, ext = os.path.splitext(part_name)
+        _, _ = os.path.splitext(part_name)
```
Comment (TomBaxter) [inline tests/providers/googledrive/test_provider.py]: Remove this line?

File: tests/providers/googledrive/test_provider.py (hunk 3 above)
Comment (cslzchen) [reply tests/providers/googledrive/test_provider.py]: I had the same thought of removing this line but not sure. I had a weak reason that we may want `splitext` to be called and not to error. However, from https://docs.python.org/3.5/library/os.path.html#os.path.splitext I see it only tries to split without doing anything interesting. I will remove this line (and another occurrence).

Comment (cslzchen) [reply tests/providers/googledrive/test_provider.py]: Complete.

File: tests/providers/googledrive/test_provider.py (hunk 6 above)
Comment (cslzchen) [reply tests/providers/googledrive/test_provider.py]: Complete.

**Pull request merged**